redbot/cogs/economy/economy.py

```diff
@@ -247,6 +247,45 @@ def __init__(self, bot: Red):
                 )
             )
 
+    @_bank.command()
+    @commands.guild_only()
+    @checks.guildowner()
+    async def removedead(self, ctx, confirmation: bool = False):
+        """Delete bank accounts for users no longer in the server"""
+        if await bank.is_global():
+            return await ctx.send(_("This command will not work in a Global bank"))
+
+        if confirmation is False:
+            await ctx.send(
+                _(
+                    "This will delete all bank accounts for user no longer in this server."
+                    "\nIf you're sure, type "
+                    "`{prefix}bank removedead yes`"
+                ).format(prefix=ctx.prefix)
+            )
+        else:
+            await bank.bank_local_clean(guild=ctx.guild)
+            await ctx.send(_("Bank accounts for user no longer in this server have been deleted."))
+
+    @_bank.command(aliases=["memberremove", "memberdel"])
+    @commands.guild_only()
+    @checks.guildowner()
+    async def memberdelete(self, ctx, member: discord.Member, confirmation: bool = False):
+        """Delete bank for specified member."""
+        if await bank.is_global():
+            return await ctx.send(_("This command will not work in a Global bank"))
+        if confirmation is False:
+            await ctx.send(
+                _(
+                    "This will delete {member} bank account."
+                    "\nIf you're sure, type "
+                    "`{prefix}bank memberdelete '{member}' yes`"
```

Comment:
Why is `{member}` wrapped in single quotes? (`'`)

Reply:
This was to provide the users with a simply Copy/Paste line for the command as some users may have spaces in their names and without the quote the command would fail

Reply:
Single quotes do not work for this like double quotes do. Single quotes are taken literally. You could use the user ID so that you do not need to worry about spaces, or swap the single quotes to double quotes and escape them.